fill/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Load a lightweight Linux image
+FROM ubuntu:22.04 AS run
+
+# Copy the build output from the GitHub Workflow to Alpine
+COPY ./fill/fill_station /bin/fill_station
+
+RUN ["chmod", "+x", "/bin/fill_station/fill_station"]
+
+# Expose the gRPC Server Port
+EXPOSE 50051
+
+# Run the binary
+ENTRYPOINT [ "/bin/fill_station/fill_station" ]
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +1 to +14]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Do you think this is the best way to do this? Instead of building the binary in a build step? Because now this Docker image can not be built locally (without some setup), it has to be built in a GitHub action.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/cornellrocketryteam/Ground-Software/24?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Unfortunately Docker can't follow bazel symlinks, so this seems like the best way for now. If we make a good local way to do this, we can have two Docker files, one for local and one for github actions.